src/cmd/linuxkit/moby/docker.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/docker/cli/cli/config/configfile"
 	distref "github.com/docker/distribution/reference"
 	"github.com/docker/docker/api/types"
+	dockertypes "github.com/docker/docker/api/types"
```

Comment:
you have still included it twice, it is the line above.

Reply:
Oh now I see it. OK thanks. Fixed.